lab-jonathan/lib/readHex.js

```diff
@@ -0,0 +1,51 @@
+let fs = require('fs');
+let oneString;
```

Comment:
You have a lot going on in this file... I'd like to see you eliminate the variable assignments in favor of directly handling values.